pkg/transformer/kubernetes/kubernetes.go

```diff
@@ -699,7 +704,12 @@ func (k *Kubernetes) Undeploy(komposeObject kobject.KomposeObject, opt kobject.C
 		return errors.Wrap(err, "k.Transform failed")
 	}
 
-	client, namespace, err := k.GetKubernetesClient()
+	client, ns, err := k.GetKubernetesClient()
+	namespace := ns
+	if opt.IsNamespaceFlag {
+		namespace = opt.Namespace
+	}
+
```

Comment:
Hey @procrypt 
You may need a rebase since Kompose shouldn't be outputting `[0000]` in the timestamp.